www-servers/nginx/patches/nginx-1.19.8.patchset

```diff
@@ -33,34 +33,34 @@ index d884487..fb487d7 100644
  
  
 diff --git a/auto/options b/auto/options
-index 521c976..b695ba6 100644
+index 80be906..8aac0bc 100644
 --- a/auto/options
 +++ b/auto/options
 @@ -13,6 +13,7 @@ NGX_CONF_PATH=
  NGX_ERROR_LOG_PATH=
  NGX_PID_PATH=
  NGX_LOCK_PATH=
-+NGX_DOC_PREFIX=
++NGX_DOC_PATH=
  NGX_USER=
  NGX_GROUP=
  NGX_BUILD=
-@@ -187,6 +188,7 @@ do
+@@ -190,6 +191,7 @@ do
          --sbin-path=*)                   NGX_SBIN_PATH="$value"     ;;
          --modules-path=*)                NGX_MODULES_PATH="$value"  ;;
          --conf-path=*)                   NGX_CONF_PATH="$value"     ;;
-+        --doc-prefix=*)                  NGX_DOC_PREFIX="$value"    ;;
++	--doc-prefix=*)                  NGX_DOC_PREFIX="$value"    ;;
```

Comment:
Ok, not sure what this should be doing? Be sure to keep the lines aligned at the same tabs, also NGX_DOC_PREFIX="$value"  is not the same as previous (changed) NGX_DOC_PATH=

Reply:
I used tab, the source uses spaces. Will fix shortly.